README.md

```diff
@@ -30,40 +31,36 @@ Currently it shows:
   * `⇡` — ahead of remote branch;
   * `⇣` — behind of remote branch;
   * `⇕` — diverged chages.
-* Mercurial repo status:
+* Current Mercurial branch and rich repo status:
   * `?` — untracked changes;
   * `+` — uncommitted changes in the index;
   * `!` — unstaged changes;
   * `✘` — deleted files;
-* Prompt character turns red if the last command exits with non-zero code.
-* Hostname only displayed when in an SSH session.
-* Username displayed only when it isn't `$LOGNAME`.
-* Username turns red when root.
-* If repo is a package, shows its version through npm (`📦`).
+* Indicator for jobs in the background (`✦`).
 * Current Node.js version, through nvm/nodenv/n (`⬢`).
 * Current Ruby version, through rvm/rbenv/chruby (`💎`).
 * Current Elixir version, through kiex/exenv/elixir (`💧`).
 * Current Swift version, through swiftenv (`🐦`).
 * Current Xcode version, through xenv (`🛠`).
 * Current Go version (`🐹`).
 * Current PHP version (`🐘`).
-* Current Rust version (`𝗥`)
-* Current version of Haskell Tool Stack (`λ`)
-* Current Julia version (`ஃ`)
+* Current Rust version (`𝗥`).
+* Current version of Haskell Tool Stack (`λ`).
+* Current Julia version (`ஃ`).
 * Current Docker version and connected machine (`🐳`).
-* Current Amazon Web Services (AWS) profile (`☁️`) ([Using named profiles](http://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html))
+* Current Amazon Web Services (AWS) profile (`☁️`) ([Using named profiles](http://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html)).
 * Current Python virtualenv.
-* Current Conda virtualenv (`🅒 `).
+* Current Conda virtualenv (`🅒`).
 * Current Python pyenv (`🐍`).
 * Current .NET SDK version, through dotnet-cli (`.NET`).
 * Current Ember.js version, through ember-cli (`🐹`).
 * Current Kubectl context (`☸️`).
+* Package version, if there's is a package in current directory (`📦`).
```

Comment:
"package" is too ambiguous, npm package?

Reply:
This section is aimed to work not only with npm-packages.